### PR TITLE
Brand Open Law Africa sites

### PR DESCRIPTION
**Pull request by nickmwangemi**
## AfricanLII
image
image

## LawLibrary
image
image

## TanzLII
image
image

## ULII
image
image

## GhaLII
image
image

## SeyLII
image
image

### CODE REVIEW

**Comment by actlikewill** [comment]
@nickmwangemi I think you can have single image in the base peachjam static folder instead of separate ones in each lii

File: ulii/static/images/open-law-africa-footer.png
Comment (longhotsummer) [inline ulii/static/images/open-law-africa-footer.png:1]: just add the images once, in the `liiweb` project.

Comment (longhotsummer) [reply ulii/static/images/open-law-africa-footer.png:1]: actually, `peachjam` makes more sense.

**Review by longhotsummer — CHANGES_REQUESTED** [review]
In general this looks good, but we need a few changes:

1. right-align the OLA logo in the headers
2. when right-aligned, the images shouldn't have `class="me-4"` because that puts margin on the right, which is not necessary
3. we'll need to do something on africanlii because on mobile the images are too wide.

The footer stuff looks good.

**Review by longhotsummer — CHANGES_REQUESTED** [review]
I've adjusted widths on mobile and removed the need for specific heights.

The OLA logos are still duplicated between liiweb and peachjam. They can move into peachjam. I think we only need two: the normal and the inverted.

Please also check with @actlikewill how to reduce them down to a decent size -- 256KB is pretty large.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**